mongodb_exporter.go

```diff
@@ -63,6 +63,7 @@ var (
 	mongodbSocketTimeout                = flag.Duration("mongodb.socket-timeout", 0, "timeout for socket operations to mongodb")
 	mongodbMaxTimeMS                    = flag.Duration("mongodb.maxtimems", 0, "maxTimeMs set for blocking database commands")
 	version                             = flag.Bool("version", false, "Print mongodb_exporter version")
+	mongos                              = flag.Bool("mongos", false, "Disable features only present in mongos")
```

Comment:
I think the comment is wrong, it should be 'disable features NOT present in mongos", rigth?